lib/fake_artist/table.ex

```diff
@@ -378,24 +422,39 @@ defmodule FakeArtist.Table do
       Logger.info(fn -> "suicide" end)
       {:stop, :shutdown, state}
     else
+      {dead_id, pids_to_ids} = Map.pop(pids_to_ids, pid)
+
       state = %{
         state
-        | connected_computers: connected_computers - 1
+        | connected_computers: connected_computers - 1,
+          pids_to_ids: pids_to_ids,
+          players: Map.delete(players, dead_id)
       }
 
+      FakeArtistWeb.Endpoint.broadcast(
+        "table:#{table_topic}",
+        "update",
+        Map.delete(state, :pids_to_ids)
+      )
```

Comment:
this makes it so players will drop off the UI when they disconnect. We can't do anything about that, but it's nice to get visual feedback while testing